tensorflow/lite/tools/build_aar.sh

```diff
@@ -41,7 +41,7 @@ function generate_list_field {
   local list_string="$2"
   local list=(${list_string//,/ })
 
-  local message+=("$name=[")
+  local message=("$name=[")
```

Comment:
Thanks for spotting this.
But does this result in build errors?

Reply:
Not a build error, my shell wouldn't run it:
`tensorflow/lite/tools/build_aar.sh: line 44: local: `message+': not a valid identifier`

Reply:
I see.